packages/block-editor/src/components/editor-skeleton/index.js

```diff
@@ -6,9 +6,24 @@ import classnames from 'classnames';
 /**
  * WordPress dependencies
  */
+import { useEffect } from '@wordpress/element';
 import { navigateRegions } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 
+function useHTMLClass( className ) {
+	useEffect( () => {
+		const element =
+			document && document.querySelector( `html:not(.${ className }` );
```

Comment:
Typo:

```suggestion
			document && document.querySelector( `html:not(.${ className })` );
```

Causing: #20504